OneCommunityGlobal/HGNRest
# Jonathan fix moving weekly summaries

# Description
Added testcases with various sample weekly summary data to debug the issue with moving weekly summaries

## Main changes explained:
- Added testcases to reporthelper.js

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. in the terminal, **cd** into src/helpers/reporthelper.js
4. input `node reporthelper.js`


## Note:
Various linting errors across multiple files that was not touched by myself
